README.md

```diff
@@ -97,326 +90,75 @@ Then, replace the *oidc* settings in `.samples.config.json` to point to your new
-
-You can read more about the `OktaAuth` configuration options here: [OpenID Connect with Okta AuthJS SDK](http://developer.okta.com/code/javascript/okta_auth_sdk#social-authentication-and-openid-connect).
-
-**Important:** When the authorization code is exchanged for an `access_token` and/or `id_token`, the tokens **must** be [validated](#validation). We'll cover that in a bit.
-
-### Custom Login Form
-To render the [Okta Sign-In Widget](http://developer.okta.com/code/javascript/okta_sign-in_widget), include a container element on the page for the widget to attach to:
-
-```html
-<!-- overview.mustache -->
-<div id="sign-in-container"></div>
-```
-
-Then, initialize the widget with the [OIDC configuration](https://github.com/okta/okta-signin-widget#openid-connect) options:
-``` javascript
-// login-custom.controller.js
-class LoginCustomController {
-  constructor(config) {
-    this.config = config;
-  }
- 
-  $onInit() {
-    const signIn = new SignIn({
-      baseUrl: this.config.oktaUrl,
-      clientId: this.config.clientId,
-      redirectUri: this.config.redirectUri,
-      authParams: {
-        issuer: this.config.issuer,
-        responseType: 'code',
-        scopes: ['openid', 'email', 'profile'],
-      },
-    });
-    signIn.renderEl({ el: '#sign-in-container' }, () => {});
-  }
-}
-```
-To perform the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1), we set the `responseType` to `code`. This returns an `access_token` and/or `id_token` through the [`/token`](http://developer.okta.com/docs/api/resources/oauth2.html#token-request) OpenID Connect endpoint.
+### Login Redirect (using [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware))
 
-**Note:** Additional configuration for the `SignIn` object is available at [OpenID Connect, OAuth 2.0, and Social Auth with Okta](https://github.com/okta/okta-signin-widget#configuration).
+> Note: The middleware library [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) is used by the Express back-end to handle the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1) 
 
-### Using a different front-end
+With AngularJS, we include the template directive `ng-click` to begin the login process. When the link is clicked, it calls the `login()` function which simply redirects the client to /login URL. We use the [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) library to handle the login and callback routes at the Express back-end.
 
-By default, this end-to-end sample ships with our [Angular 1 front-end sample](https://github.com/okta/samples-js-angular-1). To run this back-end with a different front-end:
+This middleware library constructs the login redirect URL and performs the redirect on the client's behalf.
```

Comment:
nit: Not sure we need to note this since it is specific to the front-end.

Reply:
Do you mean this is under the section "Front-end"?
We need to mention about the URL construction and redirect somewhere. I thought this was the right place to do it.

Reply:
Isn't it on line `102`? There we explain that the server is performing some "magic" to create and call the `/authorize` endpoint.

Reply:
You're right. Let me remove this section.